fe/fe-core/src/main/java/com/starrocks/consistency/ConsistencyChecker.java

```diff
@@ -459,4 +462,18 @@ public void addTabletsToCheck(List<Long> tabletIds) {
             addJob(job);
         }
     }
+
+    public void addCreatingTableId(long tableId) {
+        creatingTableIds.compute(tableId, (k, v) -> (v == null) ? 1 : v + 1);
+    }
+
+    public void deleteCreatingTableId(long tabletId) {
+        creatingTableIds.compute(tabletId, (k, v) -> {
+            if (v == null || v <= 1) {
```

Comment:
I don't think it's safe to protect the logic here with only concurrent hashmap, better using synchronized(creatingTableIds) {...}?

Reply:
![image](https://github.com/user-attachments/assets/edd78069-d1cb-4753-825d-7cbbb80f7534)
It is atomically, and I have tried.